File: web/src/components/pages/codegen/previews/trading/typescript/sdk.ts
import { createOrder } from './sdk.gen';

const { data, error } = await createOrder({
  symbol: 'AAPL',
  side: 'buy',
  type: 'limit',
  quantity: 10,
  price: 189.5,
});

if (error) {
  console.error('Order failed:', error.message);
  return;
}

console.log('Order placed:', data.id);

File: web/src/components/pages/codegen/previews/trading/python/sdk.py
from client import Client, CreateOrderData

client = Client()

order, error = client.create_order(CreateOrderData(
    symbol='AAPL',
    side='buy',
    type='limit',
    quantity=10,
    price=189.50,
))

if error:
    print(f'Order failed: {error.message}')
else:
    print(f'Order placed: {order.id}')
